applications/system/js_app/js_app.c

```diff
@@ -98,7 +98,9 @@ static void js_app_free(JsApp* app) {
 int32_t js_app(void* arg) {
     JsApp* app = js_app_alloc();
 
+#define FAP_APPID "js_app"
     FuriString* script_path = furi_string_alloc_set(APP_ASSETS_PATH());
+#undef FAP_APPID
```

Comment:
using same system as FAP_VERSION means FAP_APPID is only added to apps configured as external, from what i understand. in momentum, we have js_app as external, which would fix this. otherwise, this workaround seems to be fine

Reply:
Actually now that I think about it, this doesn't make much sense... js app doesn't come with assets, and plugins are in app data path, this line should use `EXT_PATH("apps/Scripts")`